src/plugins/console/public/application/components/editor_example.tsx

```diff
@@ -30,16 +32,27 @@ export function EditorExample(props: EditorExampleProps) {
   const elemId = `help-example-${props.panel}`;
   const inputId = `help-example-${props.panel}-input`;
 
+  const editorRef = useRef<HTMLDivElement | null>(null);
```

Comment:
Could we use a more descriptive name here? It's hard for me to reason about the difference between `editorRef` and `editorInstanceRef`.

Reply:
Sure, what do you suggest?

Reply:
Maybe `wrapperDivRef`?